src/main/java/com/prgrms/vouchermanager/domain/voucher/VoucherType.java

```diff
@@ -0,0 +1,14 @@
+package com.prgrms.vouchermanager.domain.voucher;
+
+import lombok.Getter;
+
+@Getter
+public enum VoucherType {
+    FIXED("fixed"),
+    PERCENT("precent");
+
+    private final String type = null;
+
+    VoucherType(String type) {
+    }
```

Comment:
```suggestion
    private final String type;

    VoucherType(String type) {
        this.type = type;
    }
```

null로 초기화 해야 하는 이유가 있을까요?

Reply:
type을 사용할 일이 없는데 private final String type을 선언해야하는지 궁금합니다.

Reply:
아하 사용할 일이 없으시군요!?
그렇다면 FIXED("fixed")로 선언할 필요도 없는 것 같습니다.